doc/usage/tutorial.md

```diff
@@ -2,53 +2,99 @@
 
 This doc will show how to run `kube-batch` as a kubernetes batch scheduler. It is for [master](https://github.com/kubernetes-sigs/kube-batch/tree/master) branch.
 
-## 1. Pre-condition
+## Prerequisite
 To run `kube-batch`, a Kubernetes cluster must start up. Here is a document on [Using kubeadm to Create a Cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). Additionally, for common purposes and testing and deploying on local machine, one can use Minikube. This is a document on [Running Kubernetes Locally via Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/). Besides, you can also use [kind](https://github.com/kubernetes-sigs/kind) which is a tool for running local Kubernetes clusters using Docker container "nodes".
 
 `kube-batch` need to run as a kubernetes scheduler. The next step will show how to run `kube-batch` as kubernetes scheduler quickly. Refer [Configure Multiple Schedulers](https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/) to get more details.
 
-## 2. Config kube-batch for Kubernetes
+## Install kube-batch for Kubernetes
 
-### (1) kube-batch image
-
-An official kube-batch image is provided and you can download it from [DockerHub](https://hub.docker.com/r/kubesigs/kube-batch/). The version is `v0.4.2` now.
-
-```bash
-# docker pull kubesigs/kube-batch:v0.4.2
-```
-
-### (2) Create a Kubernetes Deployment for kube-batch
-
-#### Download kube-batch
+### Download kube-batch
 
 ```bash
 # mkdir -p $GOPATH/src/github.com/kubernetes-sigs/
 # cd $GOPATH/src/github.com/kubernetes-sigs/
 # git clone http://github.com/kubernetes-sigs/kube-batch
 ```
 
-#### Deploy `kube-batch` by Helm
+### Deploy kube-batch by Helm
 
-Run the `kube-batch` as kubernetes scheduler
+Run `kube-batch` as kubernetes scheduler. The latest kube-batch version is `v0.4.2`.
```

Comment:
Do we need this “The latest kube-batch version is `v0.4.2`”? Then we need update it for every release.

Reply:
Original doc has version attached. To me, I am fine to remove this line. 

Actually I think it's better to have a stable version for users. Even though kube-batch use helm to deploy latest kube-batch version, we probably want to add tag in git clone to indicate the version.  What do you think?

```
git clone http://github.com/kubernetes-sigs/kube-batch -b v0.4.2
```

Reply:
It's fine to update doc for every release. This will guarantee users are given a stable version and guidance.